cmd/manager/main.go

```diff
@@ -59,13 +57,17 @@ type clusterConfig struct {
 	network clusternetwork.ClusterNetworkConfig
 }
 
-func printVersion() {
-	log.Info("operator", "version", version.Version)
-	log.Info("go", "version", runtime.Version(), "os", runtime.GOOS, "arch", runtime.GOARCH)
-	log.Info("operator-sdk", "version", sdkVersion.Version)
-}
-
 func main() {
+	// add version subcommand to query the operator version
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "version":
+			fmt.Printf("%s version: %q, go version: %q\n", os.Args[0], version.Get(),
+				version.GoVersion)
+			os.Exit(0)
+		}
```

Comment:
Shouldn't we fail if the users pass arguments that we don't recognize?

Reply:
Was confused by flags and sub-commands. We need a default option as @ravisantoshgudimetla mentioned. Adding it

Reply:
Not sure what this `default` option is. If you are referring to @ravisantoshgudimetla's [comment](https://github.com/openshift/windows-machine-config-operator/pull/82#discussion_r457422998). `Do we want to have long version like windows-machine-config-operator as args[0]?`, I am not sure what that means.

Reply:
Was talking about https://github.com/openshift/windows-machine-config-operator/pull/82#discussion_r457422226